lhotse/cut.py

```diff
@@ -170,6 +163,25 @@ def append(self, other: 'Cut', snr: Optional[Decibels] = None) -> 'MixedCut':
         """
         return self.overlay(other=other, offset_other_by=self.duration, snr=snr)
 
+    def pad(self, desired_duration: Seconds) -> AnyCut:
+        """
+        Return a new MixedCut, padded to `desired_seconds` duration.
+
+        :param desired_duration: The cuts minimal duration after padding.
```

Comment:
cut's, not cuts, I think

Reply:
Also perhaps should clarify that it's padding with low-energy (energy = EPS in each bin)?